nixos/hosts/lab/bitwarden/compose.bitwarden.yml

```diff
@@ -9,7 +9,7 @@ services:
       - /etc/bitwarden/settings.1.env
     # https://bitwarden.com/help/environment-variables/
     environment:
-      BW_DOMAIN: "bitwarden.ahayzen.com"
+      BW_DOMAIN: "home.hayzen.uk/bitwarden"
```

Comment:
this is likely not going to work as it should be the domain

https://bitwarden.com/help/install-and-deploy-unified-beta/#specify-environment-variables

Appears it would break nginx as it would try to put the port after the path

https://github.com/bitwarden/self-host/blob/ed2c8e2a884cd1ef41e89b0d322c15f5bcfbbd5a/docker-unified/hbs/nginx-config.hbs#L7